### PR TITLE
Create hidden page txt file

### PR DESCRIPTION
**Pull request by trv48226**
## Description

Adds the pages hidden from sitemap to two new .txt files HIDDEN_PAGES_FILE and HIDDEN_PAGES_PRODUCTION_FILE
Also adds the folders hidden from sitemap to two new .txt files HIDDEN_FOLDERS_FILE and HIDDEN_FOLDERS_PRODUCTION_FILE

Fixes # (ITS-210093)
https://jira.bethel.edu/browse/ITS-210093

## Size and Type of change

- Small Change - 1 person needs to review this

- New feature

- Config update needed

## How Has This Been Tested?

I changed the SITEMAP_BASE_FOLDER_ID to a few different Bethel pages using cms.bethel.edu and the hidden pages and folders lined up with the ones that had "hide from sitemap" checked

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have tested on multiple browsers (Chrome, Firefox, Safari, IE suite)

### CODE REVIEW

**Review by c-schwarze — CHANGES_REQUESTED** [review]
Looks good. Can we also add folders? It might be helpful to know which folders are skipped. I wonder if there is a good way to sort this. Like folders separately? Or just keep folder/pages together in the list, but stay alphebetical.

**Review by bam95899 — COMMENTED** [review]
Looks good to me.

File: sitemap.py [hunk 1]
```diff
@@ -173,3 +181,22 @@ def sitemap():
...
+    hidden = hidden_pages.split(",")
+    with open(config.HIDDEN_FILE, 'w') as file:
+        file.write("\n".join(hidden))
+    with open(config.HIDDEN_PRODUCTION_FILE, 'w') as file:
+        file.write("\n".join(hidden))
+
+    hidden = hidden_folders.split(",")
+    with open(config.HIDDEN_FOLDER, 'w') as file:
+        file.write("\n".join(hidden))
+    with open(config.HIDDEN_PRODUCTION_FOLDER, 'w') as file:
```
Comment (bam95899) [inline sitemap.py]: Might be worth it to change the config text file location variables to be something like `HIDDEN_PAGES_FILE` and `HIDDEN_FOLDERS_FILE`. Might be a bit more descriptive.

Comment (trv48226) [reply sitemap.py]: Good idea! I just added that

**Review by c-schwarze — APPROVED** [review]
Looks good to me!

**Pull request merged**